docs/topics/mta-rn-technical-preview-6-2-0.adoc

```diff
@@ -0,0 +1,9 @@
+// Module included in the following assemblies:
```

Comment:
If it does not have any technical previews (MTA rarely does) let us not include.

We need to say if there are:
- **New features and improvements**
- **Known Issues**
- **Resolved issues**

If there are **Technical Previews** or **Product Recommendations** we can include those pages but as there aren't it looks a bit weird.